src/index.ts

```diff
@@ -1,13 +1,13 @@
-import { PrismaClient } from "@prisma/client";
 
```

Comment:
1. import Elysia
2. Add route folder and put all routes there
3. You should call all these methods with the same route, let’s say for now it will be “boilerplate”